packages/rn-tester/RNTesterPods.xcodeproj/xcshareddata/xcschemes/RNTester-macOS.xcscheme

```diff
@@ -40,7 +40,9 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "NO">
+      shouldUseLaunchSchemeArgsEnv = "NO"
+      enableAddressSanitizer = "YES"
```

Comment:
Without enabling this in the xcscheme, ASAN doesn't finish initializing before the target bootstraps so tests always fail with the error below. It's not yet clear why we need this even when the same setting is enabled in the xcconfig.

`libclang_rt.asan_osx_dynamic.dylib: ==24155==ERROR: Interceptors are not working. This may be because AddressSanitizer is loaded too late (e.g. via dlopen).`